app/post_test.go

```diff
@@ -258,22 +258,17 @@ func TestUpdatePostEditAt(t *testing.T) {
 	*post = *th.BasicPost
 
 	post.IsPinned = true
-	if saved, err := th.App.UpdatePost(post, true); err != nil {
-		t.Fatal(err)
-	} else if saved.EditAt != post.EditAt {
-		t.Fatal("shouldn't have updated post.EditAt when pinning post")
-
-		*post = *saved
-	}
+	saved, err := th.App.UpdatePost(post, true)
```

Comment:
These improvements look good -- but they don't seem to be related to the PR or ticket. Can you submit these separately?